src/main/java/org/terasology/las/MountainsProvider.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.las;
+
+import org.terasology.math.TeraMath;
+import org.terasology.math.geom.BaseVector2i;
+import org.terasology.math.geom.Rect2i;
+import org.terasology.math.geom.Vector2f;
+import org.terasology.math.geom.Vector2i;
+import org.terasology.utilities.procedural.*;
+import org.terasology.world.generation.*;
+import org.terasology.world.generation.facets.SurfaceHeightFacet;
+
+@Requires(@Facet(PlayAreaFacet.class))
+@Updates(@Facet(SurfaceHeightFacet.class))
+public class MountainsProvider implements FacetProvider {
+    private Noise mountainNoise;
+//    private Noise hillNoise;
```

Comment:
If you don't want to commit your changes on `hillNoise` you can try to use `git add -p` next time and don't add the respective lines to your commit. It can also help to [`git stash`](https://git-scm.com/docs/git-stash) those changes temporarily such that they are out of the way.